sjb/config/test_cases/ci-kubernetes-conformance-node-e2e-containerized-rhel.yml

```diff
@@ -0,0 +1,59 @@
+---
+parent: 'common/test_cases/kubernetes.yml'
+overrides:
+  junit_analysis: False
+  timer: "H */2 * * *"
+extensions:
+  sync_repos:
+    - name: "aos-cd-jobs"
+  actions:
+    - type: "script"
+      title: "Install go 1.9.1"
+      script: |-
+        mkdir -p ~/bin
+        curl -sL -o ~/bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+        chmod +x ~/bin/gimme
+        gimme 1.9.1
+    - type: "script"
+      title: "Build hyperkube image"
+      script: |-
+        source ~/.gimme/envs/go1.9.1.env
+        # The docker run command run by the systemd-run can get longer than 2048 characters,
+        # causing the systemd-run command to fail. Thus, in order to minimize the
+        # length of all arguments, we need to replace /home/origin with /go prefix path.
+        # Thus, saving 54 characters (6*12-18)
```

Comment:
maybe someone more familiar with `systemd-run` can weigh in here? at first blush this is seems odd

Reply:
I have already talked about it with our systemd guys. Bug report at https://bugzilla.redhat.com/show_bug.cgi?id=1518706